pkg/cloudprovider/providers/azure/azure.go

```diff
@@ -109,6 +112,12 @@ type Config struct {
 	// Use instance metadata service where possible
 	UseInstanceMetadata bool `json:"useInstanceMetadata" yaml:"useInstanceMetadata"`
 
+	// Sku of Load Balancer and Public IP. Candidate values are: basic and standard.
+	// If not set, it will be default to basic.
+	LoadBalancerSku string `json:"loadBalancerSku" yaml:"loadBalancerSku"`
+	// ExcludeMasterFromStandardLB excludes master nodes from standard load balancer.
+	ExcludeMasterFromStandardLB bool `json:"excludeMasterFromStandardLB" yaml:"excludeMasterFromStandardLB"`
+
```

Comment:
Should assign default values to those 2 somewhere. ExcludeMasterFromStandardLB should default to true?

Reply:
ExcludeMasterFromStandardLB  is bool, so it is default to false. This works for all deployments

Reply:
Consider using pointers thus it'll be easy to change default value

ref: https://github.com/Azure/acs-engine/pull/1584#pullrequestreview-77863462